examples/README.md

```diff
@@ -0,0 +1,22 @@
+[![Powered by Flame](https://img.shields.io/badge/Powered%20by-%F0%9F%94%A5-orange.svg)](https://flame-engine.org)
+![Test](https://github.com/flame-engine/flame_example/workflows/Test/badge.svg?branch=master&event=push)
+[![Discord](https://img.shields.io/discord/509714518008528896.svg)](https://discord.gg/pxrBmy4)
+
+# flame_example
+
+This is set of small examples showcasing specific features of the Flame Engine, so it's a great source of learning for how to do certain things. [See it live here](https://flame-engine.github.io/flame_example).
+
+This app is composed of a main menu in which you can select on of the examples and play with it. Each example is a totally standalone game and is within it's own file, so you can easily checkout the code and play with it.
```

Comment:
on -> one
remove `totally` or rename to `total`
it's -> its

I think one of the `and play with it` should be removed, weird two have two sentences after each other ending with that.

Reply:
I think totally is right, it's modifying standalone, not game